model/HTMLValue.php

```diff
@@ -55,6 +54,11 @@ public function getContent() {
 			return Convert::raw2att($attrs[$matches[0]]);
 		}, $res);
 
+		// Prevent &nbsp; being encoded as literal utf-8 characters
```

Comment:
If we went the route of converting ALL utf-8 characters to the necessary entities, it would have had adverse affects on non-english locales. E.g. accented characters, asian, etc. so I've opted for the simplest solution.